Jenkinsfiles/standby-push.groovy

```diff
@@ -0,0 +1,47 @@
+stage("Announce") {
+    utils.announce_push()
+}
+
+stage("Check Pull") {
+    // Check that the image can be successfully pulled from the registry.
+    utils.ensure_pull()
+}
+
+stage("Prepare Infra") {
+    // Checkout the "mdn/infra" repo's "master" branch into the
+    // "infra" sub-directory of the current working directory.
+    utils.checkout_repo('https://github.com/mdn/infra', 'master', 'infra')
+}
+
+stage('Push') {
+    dir('infra/apps/mdn/mdn-aws/k8s') {
+        def current_revision_hash = utils.get_revision_hash()
+        withEnv(["TO_REVISION_HASH=${env.GIT_COMMIT}",
+                 "FROM_REVISION_HASH=${current_revision_hash}"]) {
+            // Run the database migrations.
+            utils.migrate_db()
+            // Start a rolling update of the Kuma-based deployments.
+            utils.rollout()
+            // Monitor the rollout until it has completed.
+            utils.monitor_rollout()
+            // Record the rollout in external services like New-Relic.
+            utils.record_rollout()
+        }
+    }
+}
+
+stage('Push EKS') {
+    def current_revision_hash = utils.get_revision_hash('prod.eks.mm')
```

Comment:
I think this needs to be moved below the `dir('infra/apps/mdn/mdn-aws/k8s') {` line below, or else it'll fail since it assumes that it's run from the `infra/apps/mdn/mdn-aws/k8s` directory.

Reply:
Good catch, fixed this in ab12738810e72d1d05fbc200d26f8f519484e5d4